great_expectations/datasource/data_connector/util.py

```diff
@@ -440,18 +445,18 @@ def list_gcs_keys(
     # Manually set to appropriate default if not already set by user
     delimiter = query_options["delimiter"]
     if delimiter is None and not recursive:
-        warnings.warn(
-            'In order to access blobs with a ConfiguredAssetGCSDataConnector, \
-            the delimiter that has been passed to gcs_options in your config cannot be empty; \
-            please note that the value is being set to the default "/" in order to work with the Google SDK.'
-        )
+        # warnings.warn(
+        #     'In order to access blobs with a ConfiguredAssetGCSDataConnector, \
+        #     the delimiter that has been passed to gcs_options in your config cannot be empty; \
+        #     please note that the value is being set to the default "/" in order to work with the Google SDK.'
+        # )
         query_options["delimiter"] = "/"
     elif delimiter is not None and recursive:
-        warnings.warn(
-            "In order to access blobs with an InferredAssetGCSDataConnector, \
-            the delimiter that has been passed to gcs_options in your config must be empty; \
-            please note that the value is being set to None in order to work with the Google SDK."
-        )
+        # warnings.warn(
+        #     "In order to access blobs with an InferredAssetGCSDataConnector, \
+        #     the delimiter that has been passed to gcs_options in your config must be empty; \
+        #     please note that the value is being set to None in order to work with the Google SDK."
+        # )
```

Comment:
Instead of removing these warnings, can we ignore instead in `pyproject.toml`? That way the warnings are still emitted (they look like valid warnings) but are not treated as errors.

Reply:
We still want warnings to be errors, but these 2 warnings are not important for now. There will be a branch that opens (after this merges) reversing any skips added.

Reply:
I guess I'm more concerned that if we merge this PR and then release tomorrow before replacing these warnings, then there will be a version of GX missing the warnings. I would feel better if these were still emitted but ignored in `pyproject.toml`.

Reply:
Please submit a code suggestion if it is that important to you

Reply:
How about something like this? The top two were already in pyproject.toml, I just moved them into this new section. 
```
# --------------------------------------- Great Expectations Intentional Warnings ----------------------------------
    # This warning is for configuring the result_format parameter at the Validator-level, which will not be persisted,
    # but is still useful for building the configuration.
    "ignore:`result_format` configured at the Validator-level will not be persisted:UserWarning",
    # This warning is for configuring the result_format parameter at the Expectation-level, which will not be persisted,
    # but is still useful for building the configuration.
    "ignore:`result_format` configured at the Expectation-level will not be persisted:UserWarning",
    # Warnings related to GCS Data Connectors
    "ignore:In order to access blobs with a ConfiguredAssetGCSDataConnector:UserWarning",
    "ignore:In order to access blobs with an InferredAssetGCSDataConnector:UserWarning",
    # --------------------------------------- End Great Expectations Intentional Warnings ------------------------------
```